### PR TITLE
handle missing station

### PR DESCRIPTION
**Pull request by Sybrand**

### CODE REVIEW

File: api/app/fireweather_bot/noon_forecasts.py [hunk 1]
```diff
@@ -46,6 +46,8 @@ def _parse_csv(temp_path: str):
     with open(temp_path, 'r') as csv_file:
         data_df = pd.read_csv(csv_file)
     station_codes = get_station_names_to_codes()
+    # drop any rows where 'display_name' is not found in the station_codes lookup:
+    data_df.drop(index=data_df[~data_df['display_name'].isin(station_codes.keys())].index, inplace=True)
```
Comment (andrea-williams) [inline api/app/fireweather_bot/noon_forecasts.py:50]: isn't this dropping all rows where the display_name is **in** station_codes ?

Comment (Kyubinhan) [reply api/app/fireweather_bot/noon_forecasts.py:50]: I am guessing `~` is a not operator in this context?

**Review by Kyubinhan — APPROVED** [review]

File: api/app/fireweather_bot/noon_forecasts.py (hunk 1 above)
Comment (andrea-williams) [reply api/app/fireweather_bot/noon_forecasts.py:50]: doh! yes, `~` is not. completely missed that

**Review by andrea-williams — APPROVED** [review]

File: api/app/fireweather_bot/noon_forecasts.py (hunk 1 above)
Comment (Sybrand) [reply api/app/fireweather_bot/noon_forecasts.py:50]: yeah - took me a while to figure out how to do a "is not in" on a dataframe.

**Pull request merged**